demo/src/static/code-markdown/trips.md

```diff
@@ -66,10 +66,10 @@ export default class HeroDemo extends Component {
 trips-layer.js:
 
 ```
-import {Layer, assembleShaders} from 'deck.gl';
 import {Model, Program, Geometry, glGetDebugInfo} from 'luma.gl';
-
-const glslify = require('glslify');
+import {Layer, assembleShaders} from 'deck.gl';
```

Comment:
`luma.gl` being a dependency of `deck.gl`, I find it more logic to put luma first.